packages/simulator/lib/dispatch/dispatchCentral.js

```diff
@@ -52,9 +52,15 @@ const getVroomPlan = async (cars, bookings) => {
 const dispatch = (cars, bookings) => {
   return cars.pipe(
     toArray(),
+    tap((cars) => {
+      if (!cars.length) {
+        warn('Fleet has no cars, dispatch is not possible.')
+      }
+    }),
+    filter((cars) => cars.length > 0),
```

Comment:
This fixes a bug that occurs if dispatch is called with 0 cars.

Reply:
Varför skapas det fleets utan bilar?

Reply:
Det kan uppstå om användaren formar fleets.json på ett sätt som leder till att en flotta blir utan bilar. Håller med om att det egentligen borde filtreras bort tidigare i flödet. Jag lägger det som en TODO.